IAST Security Controls - Compatibility Requirements (#28191)
DataDog/documentation: content/en/security/code_security/iast/security_controls/_index.md

* XPath Injection
* Cross-Site Scripting (XSS)

## Compatibility requirements

> This feature is available starting from the following versions of each language’s tracing library:

Reviewer: ```suggestion
This feature is available starting from the following versions of each language's tracing library:
```

> * **Java**: 1.45.0

Reviewer: ```suggestion
* **Java**: 1.45.0+
```

> * **.NET**: not supported
> * **Node.js**: not supported
> * **Python**: not supported

Reviewer: ```suggestion
* **.NET**: Not supported
* **Node.js**: Not supported
* **Python**: Not supported
```
